common/Windows/SecondaryWindow.cs

```diff
@@ -0,0 +1,328 @@
+                if (value != null)
+                {
+                    value.Title = title;
+                    value.TitleChanged += OnSecondaryWindowTitleChanged;
+                }
+            }
+    }
+    }
+
+    /// <summary>
+    /// Gets or sets the window content in the custom layout.
+    /// </summary>
+    /// <remarks>
+    /// This is the default content of the secondary window.
+    /// See also <seealso cref="ContentPropertyAttribute"/>.
+    /// </remarks>
+    public object SecondaryWindowContent
+    {
+        get => _windowTemplate.MainContent;
+        set => _windowTemplate.MainContent = value;
+    }
+
+    /// <summary>
+    /// Gets or sets a value indicating whether the window should use the
+    /// application theme and listen for theme changes.
+    /// </summary>
+    public bool UseAppTheme
+    {
+        get => _useAppTheme;
+        set
+        {
+            if (_useAppTheme != value)
+            {
+                _useAppTheme = value;
+                if (value)
+                {
+                    OnThemeChanged(null, ThemeSelector.Theme);
+                    ThemeSelector.ThemeChanged += OnThemeChanged;
+                }
+                else
+                {
+                    ThemeSelector.ThemeChanged -= OnThemeChanged;
+                }
+            }
+        }
+    }
+
+    /// <summary>
+    /// Gets or sets a value indicating whether the window should be modal.
+    /// </summary>
+    /// <remarks>Setting this property to true will disable interaction on the primary window.</remarks>
+    public bool IsModal
+    {
+        get => _isModal;
+        set
+        {
+            if (_isModal != value)
+            {
+                _isModal = value;
+                if (PrimaryWindow != null)
+                {
+                    // Note: This is a temporary workaround until there's a
+                    // built-in support for modal windows.
+                    PInvoke.EnableWindow((HWND)PrimaryWindow.GetWindowHandle(), !value);
+                }
+            }
+        }
+    }
+
+    /// <summary>
+    /// Gets or sets a value indicating whether the secondary window should stay on top of the primary window.
+    /// </summary>
+    public bool IsTopLevel
+    {
+        get => _isTopLevel;
+        set
+        {
+            if (_isTopLevel != value)
+            {
+                _isTopLevel = value;
+
+                // Note: Setting the owner here is a temporary workaround until there's is
+                // a built-in support for creating secondary windows with an owner while
+                // also being able to customize the content from XAML.
+                // Related: https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/windowing/windowing-overview#limitations
+                // "The Windows App SDK doesn't currently provide methods for attaching UI framework content to an AppWindow."
+
```

Comment:
We only have 3 multi-line comments in all of the Dev Home repo, and in two of those cases it's because it's around code, so that the code can be read or uncommented and used more easily. I don't believe multi-line comments like this fall under our established style.

Reply:
I'm ok with either comment style as long as we're consistent 😊 @krschau, do you suggest using `///` style? Same question for the comment outside the method. However, for the latter, the `///` is used to document the method just below the method, so maybe a multi-line style is more convenient (and it's not uncommon when we have some sort of a notification message about the code)

Reply:
@krschau, I will complete this PR, but feel free to reply on this thread and I will make a follow up PR

Reply:
I'd suggest we use /// for the documentation comments that turn into nice Intellesense popups, and // for everything else (except, admittedly, the rare case where /* */ makes something significantly easier to read or uncomment and run). The way you ended up doing it in this PR is the way I would have suggested.